File: tests/fixtures/real_index/typescript_project/src/billing.ts
/** Declares its own sessionToken, unrelated to the authentication one. */
export function sessionToken(plan: string): string {
  const total = `سلام ${plan}`
  return `billing:${total}`
}

export function openPeriod(plan: string): string {
  return `${sessionToken(plan)}:open`
}

File: tests/fixtures/real_index/python_project/app/billing.py
"""Billing service.

Declares its own session_token, unrelated to the authentication one.
"""


def session_token(plan: str) -> str:
    total = f"سلام {plan}"
    return "billing:" + total


def open_period(plan: str) -> str:
    return session_token(plan) + ":open"
